docs/index.rst

```diff
@@ -1,16 +1,13 @@
-.. include:: sections/links.rst
-
 .. toctree::
    :hidden:
 
    sections/user_guide/index
    sections/contributor_guide/index
 
-
 Unified Workflow Tools
 ======================
 
-``uwtools`` is an open-source modern Python package that helps automate common tasks needed for many standard numerical weather prediction (NWP) workflows and provides drivers to automate the configuration and execution of Unified Forecast System (UFS) components, providing flexibility, interoperability, and usability to various UFS Applications.
+``uwtools`` is an open-source modern Python package that helps automate common tasks needed for many standard numerical weather prediction (NWP) workflows and provides drivers to automate the configuration and execution of :ufs:`Unified Forecast System (UFS)<>` components, providing flexibility, interoperability, and usability to various UFS Applications.
```

Comment:
Here and below, I added a number of links that I though would be useful, and updated most existing links to use `extlinks` shortcuts.